### PR TITLE
Feat/editcard

### PR DESCRIPTION
**Pull request by PhilippWeiss85**
Dear beloved reviewer,

things that happened since the last merge:
- Added a CardDetail Component and a EditButton Component
- Added new function to unfold cards an see the DetailCard Component
- You can write stuff in the forminput of the DetailCard Component (no functionality yet expect console.logging the input data)
- fixed code of delete- and edit-button

depending user story: #9

### CODE REVIEW

File: components/CardDetail/CardDetail.js [hunk 1]
```diff
@@ -105,14 +104,32 @@ export default function CardDetails() {
       </Fieldsets>
 
       <SaveButtonDiv>
-        <Button aria-label="save details" type="submit">
+        <SubmitButton aria-label="save details" type="submit">
           Save Results
-        </Button>
+        </SubmitButton>
```
Comment (CathrinTruchan) [inline components/CardDetail/CardDetail.js:108]: Hi Phillipp, in the vercel preview this "Save-Results" Button doesn't work. Can you check if the link wrong or if something is wrong with the button?

Comment (CathrinTruchan) [reply components/CardDetail/CardDetail.js:108]: Ah I just saw, that this is probably intended? 
"You can write stuff in the forminput of the DetailCard Component (no functionality yet expect console.logging the input data)"

File: pages/_app.js [hunk 2]
```diff
@@ -1,10 +1,72 @@
...
+        court: court,
+      },
+      ...gameList,
+    ];
+    setGameList(newGameList);
+  }
+
+  function deleteCard(id) {
+    const cardListAfterDeletion = gameList.filter((game) => {
+      return game.id !== id;
```
Comment (CathrinTruchan) [inline pages/_app.js:56]: You could use the strict equality operator here ;)

File: components/CardDetail/CardDetail.js (hunk 1 above)
Comment (PhilippWeiss85) [reply components/CardDetail/CardDetail.js:108]: Hey Cathrin, yes this is "intended" :D. The save button will work with the next user story. I did not plan this well :/

**Review by CathrinTruchan — APPROVED** [review]
Good job:)

**Pull request merged**